src/backend/marsha/bbb/api.py

```diff
@@ -480,6 +482,48 @@ def recording_ready(self, request, *args, **kwargs):
             return Response(status=400)
         return Response({"message": "success"}, status=200)
 
+    @action(
+        methods=["get"],
+        detail=True,
+        url_path="invite",
+        permission_classes=[AllowAny],
+    )
+    def validate_invite_token(self, request, *args, **kwargs):
```

Comment:
maybe add throttling?

Reply:
Why not but I'm not confident with the throttling API in DRF. In the scenario, all the user accessing the classroom are on the same private network (in a university) I fear some user will not access the classroom because the quota is reached. But maybe I'm wrong.

Reply:
Not if you count only failed attempts?

Reply:
For what I understand, the throttle framework is executed before accessing the endpoint to determine if you can access it or not, like permissions. Do you now a way to count only when a condition fails ?